src/runtime/types.ts

```diff
@@ -73,6 +73,8 @@ export interface ITextPreset {
 
 export interface IOptions {
   isRTL: boolean;
+  isMobile: boolean;
+  multilingualLanguage: string;
```

Comment:
Do we need `multilingualLanguage`?

Reply:
We mentioned that on our sync meeting, also asked about that inside [issue for thunderbolt](https://github.com/wix-private/thunderbolt/issues/15951). Will remove that, if redundant.

Reply:
Removed in #35